src/services/tests/hubApi.test.ts

```diff
@@ -50,24 +50,23 @@ describe("getDocumentationHTML", () => {
       .onPost("https://api.github.com/markdown")
       .reply(200, { data: "<h2> Documentation in HTML </h2>" })
 
-    getDocumentationHTML(
+    const response = await getDocumentationHTML(
       "https://github.com/jina-ai/jina-hub/blob/master/encoders/image/BigTransferEncoder/README.md"
-    ).then((response) => {
-      expect(response).toEqual("<h2> Documentation in HTML </h2>")
-    })
+    )
+    expect(response).toEqual({ data: "<h2> Documentation in HTML </h2>" })
   })
 })
 
 describe("getHubImage", () => {
-  it("gets hub images", () => {
+  it("gets hub images", async () => {
     mockAxios
-      .onPost("https://hubapi.jina.ai/images")
+      .onGet("https://hubapi.jina.ai/images")
       .reply(200, ["Sunflowers", "Boy with an apple"])
 
-    getHubImages({ kind: ["encoder"], keywords: ["script"] }).then(
-      (response) => {
```

Comment:
Is there a reason why you changed it to the await syntax?

Reply:
better readability.

Reply:
Promises are good in general but in my opinion, it leads to a lot of unnecessary boilerplate and makes code a little bit unreadable at first go. Async-await internally works with promises. Just better sugar syntax and we can use multiple await specifically in test cases where we can use promise functions like `waitFor()` at multiple places in same test case.